src/main/java/org/folio/rest/type/StringArrayUserType.java

```diff
@@ -0,0 +1,93 @@
+package org.folio.rest.type;
+
+import java.io.Serializable;
+import java.sql.Array;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.hibernate.HibernateException;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
+import org.hibernate.usertype.UserType;
+
+public class StringArrayUserType implements UserType {
+
+  protected static final int SQLTYPE = java.sql.Types.ARRAY;
+  
+  public static final StringArrayUserType INSTANCE = new StringArrayUserType();
+
+  @Override
+  public Object assemble(final Serializable cached, final Object owner) throws HibernateException {
+    return cached;
+  }
+
+  @Override
+  public Object deepCopy(final Object o) throws HibernateException {
+    return o == null ? null : ((int[]) o).clone();
```

Comment:
Should this be a cast to String[] instead of int[]?

Reply:
Yes it should. I will update. Thanks.